dev-environment/create_environment.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/sh
 
+# Compile and package prinz if not packaged yet
+cd '..' &&
+sbt prinz/package prinz_sample/package &&
```

Comment:
use sbtwrapper as already added to our repository

Reply:
Done.